Javascript-tutorial/Shopping-site/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <div class="card">
+            <img src="assets/images/nike.webp" alt="nike">
+            <p>Nike Shoe</p>
+            <button id="purchase" onclick="purchase()">Purchase-$149</button>
+            <p id="error"></p>
```

Comment:
why is the id same for various elements? the id attribute should be unique within the entire document